README.md

```diff
@@ -127,6 +144,82 @@ OutlineView(data, children: \.children, selection: $selection) { item in
 .rowSeparatorColor(.red)
 ```
 
+### Drag & Drop
+
+#### Dragging From `OutlineView`
+
+Add the `dragDataSource` modifier to the `OutlineView` to allow dragging rows from the `OutlineView`. The `dragDataSource` takes a closure that translates a data element into an optional `NSPasteboardItem` (with a nil value meaning the row can't be dragged).
+
+```swift
+extension NSPasteboard.PasteboardType {
+  static var myPasteboardType: Self {
+    PasteboardType("MySpecialPasteboardIdentifier")
+  }
+}
+
+theOutlineView
+  .dragDataSource { item in
+    let pbItem = NSPasteboardItem()
+    pbItem.setData(item.dataRepresentation, forType: .myPasteboardType)
+    return pbItem
+  }
```

Comment:
I would prefer using full names in example code. And we don't need the article on outline view.

```suggestion
outlineView
  .dragDataSource { item in
    let pasteboardItem = NSPasteboardItem()
    pasteboardItem.setData(item.dataRepresentation, forType: .myPasteboardType)
    return pasteboardItem
  }
```

Reply:
Got it https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3